pkg/k8sclient/core_test.go

```diff
@@ -227,6 +230,9 @@ func (suite *CoreTestSuite) TestCreateClients() {
 	})
 
 	suite.Run("rg client - error case", func() {
+		suite.kubeClient = KubeClient{
+			Config: nil,
```

Comment:
Wouldn't Config be nil by default?

Reply:
It is not. Its giving the default config always